src/api/app/components/patchinfo_component.html.haml

```diff
@@ -0,0 +1,71 @@
+.pb-3
+  %h4
+    = link_to(@path) do
+      = @patchinfo['summary']
+    = category
+    = rating
+    = stopped
+    = retracted
+  %p
+    Patch created by
+    - if (packager = User.find_by_login(@patchinfo['packager']))
```

Comment:
This `if/else` statement could be simplified to `= render UserAvatarComponent.new(packager.login)` if we slightly adapt the view component to deal with inexistent users. Right now, it would crash here in this case since `avatar_object` is not referring to a user: https://github.com/openSUSE/open-build-service/blob/9190df59ecb80b20d9fba3efec29f7f4cf136a5f/src/api/app/components/user_avatar_component.rb#L15

If we don't adapt the view component, the code in this view has to change since we're fetching twice the user. Once in this view and once in the view component here: https://github.com/openSUSE/open-build-service/blob/9190df59ecb80b20d9fba3efec29f7f4cf136a5f/src/api/app/components/user_avatar_component.rb#L11